src/omero_zarr/masks.py

```diff
@@ -390,12 +401,21 @@ def masks_to_labels(
             ), f"Invalid label shape: {labels.shape}, expected {mask_shape}"
 
         fillColors: Dict[int, str] = {}
+        properties: Dict[int, Dict] = {}
+
         for count, shapes in enumerate(masks):
+            mask_value = count + 1  # Prevent zeroing
             # All shapes same color for each ROI
             for mask in shapes:
                 # Unused metadata: the{ZTC}, x, y, width, height, textValue
+                properties[mask_value] = {
+                    "omero:shapeId": mask.id.val,
+                    "omero:roiId": mask.roi.id.val,
+                }
+                if mask.textValue:
+                    properties[mask_value]["text"] = unwrap(mask.textValue)
```

Comment:
If we assume that `omero:` is a "safe"-prefix for the moment, i.e. we can do what we want there, then probably the only question I have is the naming of the `text` field. @DragaDoncila proposed `class` in https://github.com/ome/ome-zarr-py/pull/61. @will-moore, from your POV, would we post-process `text` into `class` where appropriate? Should it then be `omero:text`?

Reply:
I don't think we have to specify that this text came from OMERO in order to understand it. Similar to color - it might have come from OMERO, but we don't really care - we just want to know that the label has a color (or some text).
But these properties don't have defined keys in the spec (like flexible key-value pairs in OMERO) so I think we're "safe" to do what we want, without any prefix?
On the other hand I guess there's no harm to record the state that OMERO masks were in at the time of export, so I'm happy to use `omero:text`. I think `class` was just an example, rather than a reserved word?